src/software/ai/hl/stp/tactic/goalie/goalie_fsm.cpp

```diff
@@ -110,6 +111,39 @@ Point GoalieFSM::findGoodChipTarget(
     return chip_target;
 }
 
+bool GoalieFSM::shouldEvacuateCrease(const Update &event)
```

Comment:
are you also combining the stagnant goalie kick away in this PR as well? Update the PR description if you are.

Reply:
Accidentally included the goalie changes into this PR

I'm gonna create a separate PR for it